WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostRenderer.java

```diff
@@ -277,6 +282,23 @@ private String getPostContent() {
         return content;
     }
 
+    /*
+     * Strips inline styles from post content
+     */
+    private String removeInlineStyles(String content) {
+        if (content.length() == 0) {
+            return content;
+        }
+
+        try {
+            return Jsoup.parse(content)
```

Comment:
Nice! I tested this in case a tagless string were to get through with a length > 0 (e.g. `"    "`) and it still worked well since we should always have a `first()` body, and `toString()` on an empty list of `children()` is OK. 👍 

**Optional thought:**
- We may want to consider parsing this content [as a body fragment](https://jsoup.org/cookbook/input/parse-body-fragment) since we're not looking for document correctness. This would involve just replacing `Jsoup.parse()` with `Jsoup.parseBodyFragment()`.

Reply:
> I tested this in case a tagless string were to get through with a length > 0 (e.g. " ")
Me too, but html is funky and you never know what you're going to get. That's why I guarded jsoup with a `try/catch`. No matter how well you think you've covered all the permutations, one misplaced bracket and all goes awry. 

BTW, I did use `Jsoup.parseBodyFragment()` originally, but the resulting document still included the html/head/body tags. Ultimately it didn't make a difference whether `parse` vs `parseBodyFragment` was used. I chose `parse` 🤷  I have no issues changing it to `Jsoup.parseBodyFragment()`, as it sounds like you have more experience with the library than I do. With that said, I did find a way to remove the `children` call. So thanks for that.